lighthouse-core/gather/drivers/driver.js

```diff
@@ -492,7 +492,8 @@ class Driver {
   beginEmulation() {
     return Promise.all([
       emulation.enableNexus5X(this),
-      emulation.enableNetworkThrottling(this)
+      emulation.enableNetworkThrottling(this),
```

Comment:
should I remove `--mobile` because it's the same as disable-device-emulation but than the invert of it.